piperider_cli/profiler/event.py

```diff
@@ -72,3 +75,53 @@ def handle_column_start(self, table_name, column_result):
     def handle_column_end(self, table_name, column_result):
         print(
             f"\r    [{self.col_completed + 1}/{self.col_total}] profiling [{table_name}.{column_result['name']}] type={column_result['schema_type']} [{column_result['elapsed_milli']}ms]")
+
+
+class RichProfilerEventHandler(ProfilerEventHandler):
```

Comment:
Could this move to somewhere in the package `piperider_cli` rather than `piperider_cli.event`. We should move the display logic.